Makefile

```diff
@@ -39,6 +39,16 @@ else
 	CGO_ENABLED=0 go build $(GOFLAGS) -o "$@" "cmd/$@/main.go"
 endif
 
+.PHONY: escapes
+escapes:
```

Comment:
Can you please add a help comment here like other make targets? Context: https://github.com/confidential-containers/cloud-api-adaptor/blob/d9caf989940fd77b0d16401ced272d573f5d8760/Makefile#L20

Reply:
updated, please review.